tests/configure/test_config.py

```diff
@@ -78,3 +81,38 @@ def test_command(): # noqa
                                              TEST_PROFILE_2])
     assert '--profile can only be provided once. The profiles [{}, {}] were provided.'.format(
         TEST_PROFILE_1, TEST_PROFILE_2) in result.output
+
+
+TEST_HOST = 'https://test.cloud.databricks.com'
+TEST_TOKEN = 'testtoken'
+
+
+def test_command_headers():
+    @click.group()
+    @config.profile_option
+    def test_group():
+        pass
+
+    @click.group()
+    @config.profile_option
+    def test_group_2():
+        pass
+
+    @click.command()
+    @config.profile_option
+    @config.provide_api_client
+    def test_command(api_client): # noqa
+        click.echo(json.dumps(api_client.default_headers))
+
+    with mock.patch("databricks_cli.configure.provider.DatabricksConfig") as ConfigMock:
+        with mock.patch("uuid.uuid1") as uuid_mock:
+            ConfigMock.return_value = DatabricksConfig.from_token(TEST_HOST, TEST_TOKEN)
+            uuid_mock.return_value = '1234'
+            test_group_2.add_command(test_command, 'test')
```

Comment:
I don't understand this. Should "test_command" be consistent with "test"?

Reply:
yeah I suppose it should, I'll change it.